packages/manager/modules/adp/src/adp.service.js

```diff
@@ -0,0 +1,262 @@
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import forEach from 'lodash/forEach';
+import get from 'lodash/get';
+import isEqual from 'lodash/isEqual';
+import map from 'lodash/map';
+import set from 'lodash/set';
+
+export default class ADPService {
+  /* @ngInject */
+  constructor($q, $translate, adpConstants, CucRegionService,
+    OvhApiAdp, OvhApiCloudProject,
```

Comment:
When I try to launch it : `Unknown provider: OvhApiAdpProvider <- OvhApiAdp <- adpService` 
Maybe you forgot to bump `ovh-api-services` ?

Reply:
The `ovh-api-services` module has been updated with the new APIs (the changes have been merged to develop). You will not see these errors, once the `ovh-api-services` changes are prodded